manual/development-process/plan-story.md

```diff
@@ -0,0 +1,142 @@
+
+### Supporter
+
+This is someone on the team who has done similar work before, or who knows a lot
+about the context of a story. The aim of adding a Supporter is to make it easier
+for any developer to pick up any card, because they know who to go to for help.
+
+## Optional attributes
+
+### Edge cases
+
+Any ways the user might interact with the site, or data they might enter, that
+is different to what we expect in a straightforward case. For example, uploading
+files that cannot be identified by the file format checker.
+
+We should be thinking about edge cases during development anyway, but we've
+found that it's useful to talk about it in planning because different members of
+the team suggest different types of edge cases that an individual developer
+might not think of.
+
+### End-to-end tests
+
+Any [end-to-end tests][e2e] that should be written for this story. These tests
+are slower and more fragile than unit tests, so we should only write them if
+they provide a good signal that the main workflow is working or broken.
+
+[e2e]: https://github.com/nationalarchives/tdr-e2e-tests/
+
+### Exploratory testing
+
+Complex stories, or ones which introduce new UI or backend components, are
+likely to contain bugs that we didn't anticipate when writing automated tests.
+At planning time, we should decide whether to include time for additional
+[exploratory testing]. This might uncover bugs or just unanticipated workflows
+that we should handle before a real user tries them.
+
+Exploratory testing can be done by any member of the team who understands the
+feature being tested. Ideally it should be someone who didn't contribute much
+(or any) code to the feature, to avoid assumptions about how the application
+_should_ behave. But it's often helpful for the person testing to know something
+about the underlying components, because this can prompt them to explore
+different edge cases.
+
+[exploratory testing]: https://martinfowler.com/bliki/ExploratoryTesting.html
+
+### Cross-browser testing
+
+We don't currently have automated cross-browser testing because it would not be
+worth the effort. The site which mostly works without JavaScript, and the UI is
+based on the well-tested [GOV.UK Design System][govuk-design].
+
+When we add or modify our own components, such as the upload progress bar, we
+should test them in different browsers and on different operating systems. We
+have a Browserstack subscription managed by Digital Services.
+
+[govuk-design]: https://design-system.service.gov.uk/
+
+### Accessibility considerations
+
+We should consider whether any new components or interactions require any
+accessibility design (such as [ARIA roles][aria]) or testing once the feature
+is complete.
+
+[aria]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
+
+### Security considerations
+
+We should think about whether a new feature could potentially introducing
```

Comment:
Should read: "... whether a new feature could potentially **introduce** security ..."

Reply:
Fixed, thanks.